GVFS/GVFS.Installer.Mac/scripts/Distribution.xml

```diff
@@ -34,7 +34,7 @@
         
         function GetBlockingProcesses()
         {
-            var watchList = [ "gvfs", "GVFS.Mount", "git", "gitk", "Wish" ];
+            var watchList = [ "GVFS.Mount", "git", "gitk", "Wish" ];
```

Comment:
If we are going to run upgrade "in-line" - then the `gvfs upgrade` verb will be running while the installer is running. If `gvfs` is in the watchlist, then upgrade will be blocked if the `gvfs upgrade` command is running.

For now, I removed `gvfs` from the list of blocked processes - active mount processes will still block the installer (which seems a bit unexpected as well - I would expect the installer to unmount as part of running).

I thought about whether we need to be more restrictive in the future, maybe only allow the expected gvfs upgrade command? but this seems like it could be fragile.